benchmark/net/net-is-ip-v4.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const common = require('../common.js');
+const { isIPv4 } = require('net');
+
+const minIPv4 = '0.0.0.0';
+const maxIPv4 = '255.255.255.255';
+const invalid = '0.0.0.0.0';
```

Comment:
This should instead be an array of various IPs that includes more kinds to improve coverage (in the case of the current implementation: to hit the alternate patterns in the regexp).

Reply:
sure, I am open for suggestions. Looking forward for them.

Reply:
Here are a couple of concrete suggestions if you need some:

* 192.168.0.1
* 10.168.209.250

Reply:
What do you mean regarding an array? Is there a specific way how you do this? Or just iterating through the array

Reply:
> What do you mean regarding an array? Is there a specific way how you do this? Or just iterating through the array

```js
const ips = [
  '0.0.0.0',
  '255.255.255.255',
  '0.0.0.0.0',
  '192.168.0.1',
  '10.168.209.250',
];

// ...

for (let i = 0; i < n; ++i) {
  for (let j = 0; j < ips.length; ++j)
    isIPv4(ips[j]);
}
```

Reply:
implemented as requested